modules/ROOT/pages/shared-bucket-access.adoc

```diff
@@ -3,15 +3,15 @@
 == Shared Bucket Access
 :pageOriginRel: "sg=1.5, Cbs=5.0)_"
 
-The objective of Mobile-Web Data Sync is to ensure that the security and replication metadata needed by mobile applications is preserved.
-
 Sync Gateway uses the _Shared Bucket Access_ feature to synchronize document changes made through Couchbase SDKs and N1QL queries to be synchronized to Couchbase Lite clients, and vice versa.
-
 The following diagram shows different clients (mobile, web, desktop) writing data to the same bucket in Couchbase Server.
-Changes were made by James, but itimage::shared-bucket-access.png[]
 
-The metadata created by the Sync Gateway is abstracted from applications reading and writing data directly to Couchbase Server.
-Sync Gateway does this by utilizing a Couchbase Server feature called XATTRs (e**X**tended **ATTR**ibutes) to store that metadata into an external document fragment.
+image::shared-bucket-access.png[]
+
+In previous releases, you either had to ensure all writes happened through Sync Gateway, or had to set up bucket shadowing to ensure that the security and replication metadata needed by mobile applications was preserved.
+
+In this release, the metadata created by the Sync Gateway is abstracted from applications reading and writing data directly to Couchbase Server.
+Sync Gateway 1.5 utilizes a new feature of Couchbase Server 5.0 called XATTRs (e**X**tended **ATTR**ibutes) to store that metadata into an external document fragment.
```

Comment:
```suggestion
Sync Gateway a feature of Couchbase Server 5.0 called XATTRs (e**X**tended **ATTR**ibutes) to store that metadata into an external document fragment instead of inline within the document body.
```